eyelid-match-ops/src/encoded.rs

```diff
@@ -24,18 +24,18 @@ pub mod test;
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct PolyCode<C: EncodeConf> {
     /// The polynomials, encoding one block of rows each. Storage variant.
-    polys: Vec<Poly<C::PlainConf>>,
+    pub polys: Vec<Poly<C::PlainConf>>,
```

Comment:
Usually we'd add read-only `polys() -> Vec<Poly<C::PlainConf>>` and `masks()` methods, because they only allow outside code to read the value. Using `pub` means that outside code can modify these values.

But I'm happy to just make that into a TODO comment.

```suggestion
    //
    // TODO: use read-only accessor methods instead of `pub` for all 4 fields in these 2 structs.
    pub polys: Vec<Poly<C::PlainConf>>,
```